packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -467,6 +467,24 @@ const model = ref<Model | null>(null);
 const mmt = ref<MiraModel>(emptyMiraModel());
 const mmtParams = ref<MiraTemplateParams>({});
 
+const configuredMmt = computed(() => {
+	const mmtCopy = cloneDeep(mmt.value);
+	if (!mmtCopy) return mmtCopy;
```

Comment:
Not exactly sure what you are trying to say here??? `if (!a) return a` ?

Reply:
oof my bad